cedar-policy/src/api.rs

```diff
@@ -2688,7 +2040,7 @@ fn is_static_or_link(
 
 /// Like [`itertools::Itertools::partition_map`], but accepts a function that can fail.
 /// The first invocation of `f` that fails causes the whole computation to fail
-fn fold_partition<T, A, B, E>(
+pub(crate) fn fold_partition<T, A, B, E>(
```

Comment:
This was recently introduced -- I'm wondering if it's actually used in more than one file?  Or if we can just move it to the place it is used.

Reply:
It's currently used (indirectly) by `PolicySet::to_json` and in the tests. One option is to move the relevant tests for this function back to api.rs, how do you feel about that?

Reply:
No strong feelings either way.